src/SmartComponents/SystemInventory/SystemInventoryCard.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
```

Comment:
we do this everywhere... any qualms with killing it globally in the app?

Reply:
Fine by me, you can disable this rule and have it to warning instead.

Reply:
Heck yeah, i can do that in a followup, no need to add it here 🤗